libs/s25main/ingameWindows/IngameWindow.h

```diff
@@ -104,6 +104,8 @@ class IngameWindow : public Window
 
 protected:
     void Draw_() override;
```

Comment:
should Draw_ be final here then?

Reply:
I can still imagine some Windows wanting to override this hence I left it. But I just checked and we don't used that right now, so made final.

BTW: The Jenkins build didn't seem to start. Do you know why?

Reply:
the jenkins cloud plugin was changed I had no time yet to fix the configuration :-/